Remember last position

bequiett opened the pull request:
마지막으로 프로그램이 종료된 위치를 저장해 다음 실행 시 저장된 위치에 프로그램을 표시합니다.

Commit:
Remember Last Program Pos

Memorizes last program's position to registry and brings the program back to the saved position.

Commit:
Using Properties

wowvv0w commented:
설정 저장 방식을 json에서 Properties.Settings로 바꾼 후 merge 하겠습니다

wowvv0w commented:
원래 버그인줄 알았던 설정 저장 적용이 안되는 현상이 단순 사용자 실수였다는 것을 알고나서, json으로 변경했던 설정 저장 방식을 Properties.Settings로 다시 되돌리려고 했습니다.
하지만 다음과 같은 이유로 json을 계속 사용하려고 합니다.

1. 파일 이름 또는 경로가 변경되면 새로운 해시값이 할당되어 설정 유지가 안됨, 강력한 이름이나 SettingsProvider 같은 방법도 있으나 너무 복잡해짐
2. 추후 구현할 프리셋: 프로그램의 새로운 버전이 나왔을때 폴더 복사 붙여넣기로 사용자가 설정한 프리셋을 그대로 사용 가능하도록 할 계획인데, 그때 폴더 내에 설정 파일이 들어 있다면 더욱 편할 것이라 생각함
3. 사용자에게 문제 발생 시 설정 파일 확인을 쉽게 하도록 하기 위함

그래서 부탁드리는 것인데, Top, Left 값을 기존 방식에서 Data￦Config.json으로 옮겨서 구현해주실 수 있으십니까? 위 json 파일은 최근 커밋에 있습니다. 여유가 없으시다면 일단 merge 후 제가 수정하도록 하겠습니다.

bequiett commented:
내일 중으로 변경하겠습니다.

bequiett committed:
Merge branch 'wowvv0w:main' into remember-pos

Commit:
Using Config.json

Saves program's last position on Config.json with in the form of double array.

Commit:
Minor Changes

Saves position as 0 if current position is negative.

Commit:
Stability Update

Fixed crashing when position value in Config.json was modified to null value by user.

bequiett committed:
Update MainViewModel.cs

bequiett committed:
Update MainViewModel.cs
